### ISSUE TITLE
This is confusing to use - Whats the idiots guide to converting a docx to tex?

### ISSUE THREAD

**Issue by NathanKennedy** [transpect/docx2tex#60]
This is confusing to use - Whats the idiots guide to converting a docx to tex?

I use Linux and there is no guide how to use this? Does it even have a interface or isit all done in the command window? 

Gives examples or no one can use this please...

Also it says you need java 1.7 to 1.14 to run on linux - can you link what we need because I've heard the name Java but no idea what it does or how to use it.

**Comment by gimsieke** [comment]
Please don’t post an issue when the answer is in plain sight [on this repo’s start page](https://github.com/transpect/docx2tex#linuxmacosx). And please don’t use such a language of entitlement. Some basic IT literacy is required in order to use it, for example how to install Java on Linux. These are things that can be easily looked up [using a search engine](http://letmegooglethat.com/?q=java+linux). In particular if you are going to use LaTeX, some familiarity with command line programs can be expected.

**Comment by gimsieke** [comment]
We might occasionally offer a Web service where you can upload your docx, wait a couple minutes and then download a LaTeX, maybe also a BibTeX, and a PDF file.

**Comment by NathanKennedy** [comment]
I edited my post because I didnt realise it caused offence my apologies! I have been looking through google to find a program that can change my word document to Latex. The document has a lot of heavy duty word formulas.

./d2t [options ...] myfile.docx
If my file is in the documents folder, how do I use this command on it? I use linux but rarely use command prompt.

**Comment by NathanKennedy** [comment]
> We might occasionally offer a Web service where you can upload your docx, wait a couple minutes and then download a LaTeX, maybe also a BibTeX, and a PDF file.

I edited my post because I didnt realise it caused offence my apologies! I have been looking through google to find a program that can change my word document to Latex and its very frustrating finding something that may work and theres no clear instructions so I spent like an hour trying to work through what you have written to no avail. My document has a lot of heavy duty word formulas so desperate to get it on my machine as I have switched from windows to linux.

./d2t [options ...] myfile.docx
If my file is in the documents folder, how do I use this command on it? I use linux but rarely use command prompt.
